packages/sitecore-jss-react/src/enhancers/withEmptyFieldEditingComponent.test.tsx

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable no-unused-expressions */
+import React, { forwardRef } from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import { withEmptyFieldEditingComponent } from './withEmptyFieldEditingComponent';
+import { DefaultEmptyFieldEditingComponentText } from '../components/DefaultEmptyFieldEditingComponents';
+import { describe } from 'node:test';
+
+describe('withEmptyFieldEditingComponent', () => {
+  const testMetadata = {
+    contextItem: {
+      id: '{09A07660-6834-476C-B93B-584248D3003B}',
+      language: 'en',
+      revision: 'a0b36ce0a7db49418edf90eb9621e145',
+      version: 1,
+    },
+    fieldId: '{414061F4-FBB1-4591-BC37-BFFA67F745EB}',
+    fieldType: 'single-line',
+    rawValue: 'Test1',
+  };
+
+  type TestComponentProps = {
+    field?: {
+      value?: string;
+      metadata?: { [key: string]: unknown };
+    };
+    editable?: boolean;
+  };
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const TestComponent = (props: TestComponentProps) => {
+    return (
+      <div>
+        <h1>{props.field?.value}</h1>
+        <h2>foo</h2>
+        <p>bar</p>
+      </div>
+    );
+  };
+
+  // eslint-disable-next-line react/display-name
+  const TestComponentWithRef = forwardRef(
+    (props: TestComponentProps, ref: React.ForwardedRef<HTMLDivElement>) => {
+      return (
+        <div>
+          <h1>{props.field?.value}</h1>
+          <h2 ref={ref}>foo</h2>
+          <p>bar</p>
+        </div>
+      );
+    }
+  );
+
+  it('Should render provided default empty value placeholder component if field value is empty in metadata edit mode', () => {
```

Comment:
Let's add a _describe_ block called _Metadata mode_, so you can avoid adding this in a title of each test case